### ISSUE TITLE
Permanently hide the web page footer

### ISSUE THREAD

**Issue by nickpapag** [Ylianst/MeshCentral#744]
Hi Ylian and team, I Love Mesh Central, thanks for all your work! We currently have this deployed for >100 computers with about a dozen groups and it works great. I'm wondering if you could add a expand / collapse feature for the Groups under My Devices. With the large number of devices, I find myself endlessly scrolling up an down as I connect to different devices especially on mobile. Basically, if you could have the list of devices collapsed by default so just the group name was displayed, then clicking the name could expand / collapse the device list for the group. I'm usually only working in 1-2 groups at a time so having the others collapsed to just their name would make things much more efficient.

Also, could you possibly move the Terms & Privacy link from the bottom right to the bottom left, or somewhere else and then remove the blue bar across the bottom? that would give us some more real estate when remoting into another system.

Thanks again, this is absolutely amazing!

**Comment by OutbackMatt** [comment]
Top right of screen when at a desktop there is a four headed arrow pointing obliquely to the corners 
Clicking on this maximises the available real estate.

You can the ALSO make your browser full screen
(In Chrome for instance>> Top right = three dots vertically aligned >> square dashed outline next to zoom control)

**Comment by MailYouLater** [comment]
It's generally best to keep separate bugs and/or feature requests separate, opening a separate issue for each one.  The exception to this general rule is when an issue is so closely related that there's no point in opening a second one because both should be tracked together (e.g. when one feature has multiple problems that seem likely to be closely related) but this isn't one of those cases.

Regarding hiding the footer, add `?hide=4` to the end of your URL.  I don't think moving the Terms & Privacy link to a different location is supported at the moment, but as a workaround you could easily add it to the top by changing the `Title2` setting in your `config.json` to include `<a href="/terms">Terms & Privacy</a>`.

Regarding the feature request to make the device groups collapsible, I don't think having them default to being collapsed is a good idea unless the default is configurable, but I think having a "Collapse All"/"Expand All" button at the top would be a good addition to this request, and would give @nickpapag the functionality they're requesting without potentially confusing people who expect their devices to just be visible, like they are currently.

**Comment by Ylianst** [comment]
I am working on the footer thing now and needed to fix a lot of things with the ?hide=x feature. Hopefully with have a new version tomorrow with improvements.

**Comment by Ylianst** [comment]
Published MeshCentral v0.4.6-b with UI improvements including many improvements to the ?hide=x feature. If you do add ?hide=4, it hides the footer, ?hide=1 will hide the header and ?hide=5 will hide both. With this new version, if you hide the header, the logout indicator moved to the toolbar. So, this is very compact.

![MC2-UIHide](https://user-images.githubusercontent.com/1319013/71295880-9f874200-2332-11ea-93b3-44a3b15d0202.png)

Also, if you want your web site to always have some hidden UI elements, you can now add "hide":x in the domain section of the config.json, for example:

```
  "domains": {
    "": {
      "title": "MyServer",
      "hide": 4
    }
  }
```

This would always hide the footer. Let me know if this works.

**Comment by Ylianst** [comment]
FYI. I am going to consider this the hidden footer issue. You can open a new issue for device grouping if you like. Thanks.

**Comment by OutbackMatt** [comment]
The 'hide' works for me thanks

**Comment by Ylianst** [comment]
Just published MeshCentral v0.4.6-h where users can now toggle the footer bar and hide it for more screen space. If you put hide=4, it will hide this option as the footer will always be hidden. Feedback appreciated.

![MC2-FooterToggle](https://user-images.githubusercontent.com/1319013/71315230-ad10fa80-240c-11ea-89ae-a90bcb35661a.png)

**Comment by Ylianst** [comment]
Going to close this one as I think it's fixed in many different ways. Please open a new issue if something is not right.
